src/MicrosoftClientAcceptanceTests/MicrosoftClientAcceptanceTests.csproj

```diff
@@ -12,14 +12,14 @@
   <ItemGroup>
     <PackageReference Include="NServiceBus.AcceptanceTesting" Version="8.0.0" />
     <PackageReference Include="NServiceBus.Gateway.AcceptanceTests.Sources" Version="4.0.0" />
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="3.0.1" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="3.1.5" />
```

Comment:
This was included because of a CVE? It isn't listed in the things to modify.

If it is because of the CVE we need to take note of that so that it gets released.

Reply:
This does not actually need to be released. The SqlClient references are only in test projects. NServiceBus.Gateway.Sql uses a connection builder pattern like SQL Persistence and [the component itself](https://github.com/Particular/NServiceBus.Gateway.Sql/blob/release-2.0/src/NServiceBus.Gateway.Sql/NServiceBus.Gateway.Sql.csproj#L10-L12) does not directly reference any SqlClient package.